server/lib/config.js

```diff
@@ -1,16 +1,24 @@
 const settings = { };
 let currentProvider = null;
 
+const boolify = (item) => {
+  if (item === 'true' || item === 'false') {
+    return item === 'true';
```

Comment:
I'm confused by this logic. why isn't it just this?

```
const boolify = (item) => item === 'true' || item === true;
```

Reply:
we're trying to `boolify` everything,  so if we're trying to boolify `AUTH0_CLIENT_ID`, we should get client_id and not `false`